### PR TITLE
Handle first node template failed

### PR DESCRIPTION
**Pull request by galuhest**

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -28,12 +28,11 @@
 default['elasticsearch']['routing_allocation_disk_watermark_low_threshold'] = "100gb"
 default['elasticsearch']['routing_allocation_disk_watermark_high_threshold'] = "50gb"
 default['elasticsearch']['routing_allocation_disk_watermark_flood_stage_threshold'] = "10gb"
-default['elasticsearch']['node_awareness_value'] = "$HOSTNAME"
-default['elasticsearch']['node_awareness_attribute'] = 'hostname'
 
 # ES 7.x
 default['elasticsearch']['initial_master_nodes'] = 'elasticsearch.service.consul'
 default['elasticsearch']['discovery_seed_hosts'] = 'elasticsearch.service.consul'
+default['elasticsearch']['discovery_type'] = 'multiple-node'
```
Comment (fadlinurhasan) [inline attributes/default.rb:35]: so we will have to set `discovery_type` for small cluster on our barito, right? please note this or adjust directly on our cluster_templates.

**Review by fadlinurhasan — COMMENTED** [review]
Please bump the version and adjust the changelog.
we can have a version based on es version or cookbook changes itself, wdyt?

File: attributes/default.rb (hunk 1 above)
Comment (galuhest) [reply attributes/default.rb:35]: Correct, i follow Elasticsearch by setting the `discovery_type` as multiple-node by default. Single node cluster will need to set this value to `single-node`. Where should i put the documentation on this?

Comment (fadlinurhasan) [reply attributes/default.rb:35]: we can start documented this on our jira. We can have explanation per each component or per cluster type.

**Review by fadlinurhasan — APPROVED** [review]

**Pull request merged**